api-gateway-config/conf.d/tracking_api.conf

```diff
@@ -3,7 +3,11 @@
 # This VHost should use a server_name matching the hostname
 
 server {
-  listen 5000 default_server;
+  listen 5000;
+  server_name default_server;
+  access_log /var/log/api-gateway/tracking_access.log;
```

Comment:
Logging to stdout avoids filling up all the disk space and causing an out of space issue in containers.